clerk/backend/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "backend"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+actix-web = "4.4.1"
+clerk-rs = "0.2.3"
+dotenv = "0.15.0"
+openssl-sys = { version = "0.9.9", features = ["vendored"] }
```

Comment:
is this one needed?

Reply:
On my windows machine I am having a open ssl issue so adding fixes that

Reply:
The clerk lib is using reqwest with native TLS and no feature flag to enable it. We could suggest that as an improvement to the library.

Reply:
I was not knowing that. Great to see that coming up!!